tensorflow/python/ops/parallel_for/pfor.py

```diff
@@ -1593,16 +1593,14 @@ def _add_to_stack(x):
           else:
             converter = _pfor_converter_registry.get(y_op.type, None)
           if converter is None:
+            root_cause = (f"there is no register converter for this op.")
```

Comment:
Let me know how we want to extend this message: 

- Do we want to ask to open a FR ticket/? 
- Do we want to ask to contribute a PR?
  -  Do we have a bare minimal documentation to share on how to implement a converter?